python/ray/util/sgd/BUILD

```diff
@@ -90,6 +92,16 @@ py_test(
     tags = ["team:ml", "exclusive", "pytorch"],
     deps = [":sgd_lib"],
 )
+
+py_test_module_list(
```

Comment:
@amogkam Could you take a look later on to see whether we want to enable them or not?

Reply:
Removed.